privacy/views/data_protection_menu_view.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
+<!-- Copyright 2018 Eficent Business and IT Consulting Services S.L.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0) -->
+<odoo>
+<data>
+
+    <record id="action_data_protection_partner_form" model="ir.actions.act_window">
+        <field name="name">Partners</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">tree,form</field>
+        <field name="domain">[]</field>
+        <field name="filter" eval="True"/>
+    </record>
+
+    <menuitem id="parent_menu_data_protection"
+              name="Data Protection"
```

Comment:
Thanks for these modules on this topic. Just an idea but to have concise name on top menu, is it possible to rename to a shorter one like 'Data' or 'privacy'. 'Data protection' could be added to a sub menu.

In community main menu, it could be better to have short name to see the whole bar.
What do you think ?

Thanks

Reply:
I'm OK with "Privacy".
It should be a change on 9.0, that would be then forward ported to 10 and 11.
We also need a volunteer to contribute with an icon 😏 .